helpers/perfMonitor.js

```diff
@@ -0,0 +1,25 @@
+export class PerfMonitor {
+
+	constructor(host) {
+		this.constructedTime = window.performance.now();
+		this.host = host;
+	}
+
+	hostConnected() {
```

Comment:
Is this being used?  The PR description mentions "connected" but it looks like the "constructed" time is being used in the calc below... which one did you want to use?

Reply:
Ah you're right. Technically we could use this instead of constructed time... which... yeah I'm not sure which one would be more accurate. I'll remove this for now.